qiita_db/util.py

```diff
@@ -43,7 +43,7 @@
 from __future__ import division
 from future.builtins import zip
 from future.utils import viewitems
-from random import choice
+from random import SystemRandom
```

Comment:
There is a warning in the python documentation that said that random should not be used for security purposes, and it recommends the usage of SystemRandom. Since now I'm using the generate random string function to generate the client id / client secret pair for oauth2, I think is better to use SystemRandom.